templates/base.html

```diff
@@ -58,6 +60,25 @@
                 <span class="icon-bar"></span>
               </button>
               <a class="navbar-brand" href="alerts.html">Talos Alert Manager</a>
+
+                        <select class="form-control" id="actions" onchange="performAction()">
+                                <option selected>Actions</option>
+                                <option >Change Revision</option>
+                                <option>Add Bug</option>
+                                <option>Add Comment</option>
+                                <option>NEW</option>
+                                <option>Back Filling</option>
+                                <option>Investigating</option>
+                                <option>Resolved</option>
+                                <option>Shipped</option>
+                                <option>Wont Fix</option>
+                                <option>False Alarm</option>
+                                <option>Ignore</option>
+                                <option>Not Tracking</option>
+                                <option>Too Low</option>
```

Comment:
yeah they will change automatically and are similar to changing status for each alert by selecting options from that particular row